### ISSUE TITLE
Default values for aliases don't function as expected

### ISSUE THREAD

**Issue by chrisleavoy** [StackStorm/hubot-stackstorm#51]
aliases/roll.yaml:
```yaml
---
name: "roll"
action_ref: "ansible.playbook"
description: "Perform a rolling deployment with Ansible playbook"
formats:
  - "roll1 {{playbook=deploy.yaml}} {{cwd=/opt/server-tooling/rollin}} {{inventory-file=test}} {{limit=10.2.30.121}}"
```
Sending:
```
bob roll1 playbook=deploy.yaml cwd=/opt/server-tooling/rollin inventory-file=test
```

Results in:
```
Action ansible.playbook completed.
status : failed
execution: 56391f6b02ebd505aa3bc807

result :
--------
return_code : 1
stderr : ERROR: Specified --limit does not match any hosts
Executed command "/opt/stackstorm/virtualenvs/ansible/bin/ansible-playbook --limit=inventory-file=test --inventory-file=test deploy.yaml"
```

Notice how it clobbered the the limit parameter and didn't inject the default.

**Comment by emedvedev** [comment]
Hi Chris,
Sorry it took us a lot of time to look into the issue. We're working on a major chatops overhaul, and as soon as we release the new version of StackStorm, this will be fixed.
This could work as a temporary solution:
```
roll1 deploy.yaml /opt/server-tooling/rollin test
```

**Comment by emedvedev** [comment]
That said, Ansible pack still has to be reworked.

**Comment by emedvedev** [comment]
Fixed as a part of 1.2.0. Note the changes in Ansible alias parameters.
